src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2652,23 +2654,31 @@ The Signal API is available inside ProActive Tasks through the *signalapi* <<_gl
 
 It is easy to use, and offers a set of basic operations on signals summarized as follows:
 
- * *readyForSignal*: declares that the current job is ready to receive and handle one or more signals.
+ * *readyForSignal*: declares that the current job is ready to receive and handle one or more signals with or without variables.
    This operation constructs new signals (by adding the prefix `ready_` to the input signals), then adds them to job signals.
    For example, if a job declares ready for the signal `soft_stop`, the signal `ready_soft_stop` is added to the set of job signals.
    Additionally, if the input signal (e.g., `soft_stop`) already exists among the job signals (which means that it has been sent and handled previously),
     the operation ready for signal automatically removes it, thus the job is ready again to receive and handle the same signal.
 
- * *sendSignal*, *sendManySignals*: adds one or more signals given as input to the current job signals.
+ * *sendSignal*, *sendManySignals*: adds one or more signals with or without variables given as input to the current job signals.
```

Comment:
adds => send programmatically

You should add "Though signals are meant to be sent manually by a human user through to ProActive web portals, it is also possible to use these methods to send them programmatically inside the workflow).